CHANGELOG.yaml

```diff
@@ -1,3 +1,8 @@
+master:
+    new features:
+     - >-
+        GH-2509 Added a new option `skip-executions-summary` in newman to skips addition of request,     response to summary executions report generated for specified reporter.
+
```

Comment:
Seems to be some formatting issues here. Lots of space before `response to summary...`

Reply:
Fixed